README.md

```diff
@@ -221,6 +218,7 @@ Structure of the state file:
 }
 ```
 
+### Additional options for minting and target
```

Comment:
I guess it should be capitalized, like the other headlines
Also the wording here is a bit odd: `Additional options for ... target`
Maybe makes sense to make `Minting Manually` and `Use a Raiden Full Node` a 3rd level header

Reply:
@nephix Thanks for your feedback

Looking at it again I would just remove the two sections (Minting manually and use a Raiden Full Node) as it is our main readme and I want to keep it short and precise. @taleldayekh Are you ok with it?

Reply:
Talked with @taleldayekh and we agreed to remove it. I am implementing this now.